test/integration/targets/vmware_guest/tasks/multiple_disk_controllers_d1_c1_f0.yml

```diff
@@ -0,0 +1,242 @@
+- name: Wait for Flask controller to come up online
+  wait_for:
+    host: "{{ vcsim }}"
```

Comment:
Hi,

We recently did a refactoring of the functional test-suite, the goal is to be able to run it on a regular VMware environment, this instead of vcsim.
As a consequence, you should not handle vcsim in the functional test anymore, use the `prepare_vmware_roles` role instead. e.g: https://github.com/ansible/ansible/blob/devel/test/integration/targets/vmware_drs_group/tasks/main.yml

You should also ensure the resources that are created during your test are properly removed at the end.

Reply:
Thanks for reviewing, I've rewrite the test case.